serverless/integration/integration_test.go

```diff
@@ -93,6 +96,32 @@ func RunIntegration(t *testing.T, s log.Storage, f client.Fetcher, lh *rfc6962.H
 			t.Fatalf("Failed to update tracked log state: %q", err)
 		}
 		newCheckpoint := lst.LatestConsistent
+		// Verify that the returned checkpoint note is as expected.
```

Comment:
Can you just save the `cpNote` (and perhaps `cpNoteSigned`) from L84/L85 and compare them with what you get back from `Update`?

It seems like a lot of the testing below is recovering things which should already be covered by tests in the `log` package (around the correct form of a `log.Checkpoint`, etc.)